Casks/t/thebrowsercompany-dia@beta.rb

```diff
@@ -0,0 +1,28 @@
+cask "thebrowsercompany-dia@beta" do
+  version "0.32.2"
+  sha256 :no_check
+
+  url "https://releases.diabrowser.com/release/Dia-latest.dmg"
+  name "Dia"
+  desc "A smart browser where you can chat, write, learn, plan and shop with your tabs"
+  homepage "https://www.diabrowser.com/"
+
+  livecheck do
+    url :url
+    regex(/Dia-(\d+(?:\.\d+)+)-\d+\.dmg/i)
+    strategy :header_match
+  end
+
+  auto_updates true
+  depends_on macos: ">= :sonoma"
```

Comment:
```suggestion
  depends_on arch: :arm64
  depends_on macos: ">= :sonoma"
```

It seems that only binaries for ARM architectures are available.